plantseg/viewer/widget/utils.py

```diff
@@ -70,6 +70,54 @@ def on_done(result):
     return future
 
 
+def start_prediction_threading_process(func: Callable,
```

Comment:
minor: rename to either `start_prediction_thread` or `start_prediction_process` but don't use both